nion/swift/LinePlotCanvasItem.py

```diff
@@ -420,8 +420,10 @@ def convert_to_calibrated_size_str(f: float) -> str:
 
     def __view_to_intervals(self, data_and_metadata: DataAndMetadata.DataAndMetadata, intervals: typing.List[typing.Tuple[float, float]]) -> None:
         """Change the view to encompass the channels and data represented by the given intervals."""
+        # X dimensional factors are controlled by dimensional_calibrations
```

Comment:
I don't think these comments belong here. They're a general knowledge about the calibrations/line plot and I think they're just extra here since they could literally be applied to almost any function in this file.